tensorrt_llm/_torch/pyexecutor/py_executor.py

```
@@ -965,6 +976,17 @@ def _kv_connector_start_batch(self, scheduled_batch):
             self.kv_connector_manager.worker.start_load_kv(
                 torch.cuda.current_stream())
 
+    def _kv_connector_refresh_unfinished_tasks(self):
+        if not self.use_flexkv:
+            return
+        if len(self.active_requests) == 0:
+            return
+        if not self.kv_connector_manager:
+            return
+        logger.warning(f"No scheduled requests, but flexkv have pending put requests")
```

Ruff (0.14.5)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)